packages/client/src/hooks/useNotifications.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  nextNotification,
+  evictNotification,
+} from "@/store/actions/notificationsActions";
+import { getActiveNotification } from "@/store/selectors/notifications";
+import { NotificationInterface } from "@/types/store";
+import { useEffect, useRef } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+interface useNotificationsParams {
+  timeouts: {
+    minTimeout?: number;
+    maxTimeout?: number;
+  };
+}
```

Comment:
This also works, don't get me wrong, but it's ok to lift is one level:
instead of having 
```typescript
params: {
  timeouts: {
    // min and max timeout
  }
}
```

we can simply use 
```typecript
timeouts: {
  // min and max timeout
}
```